library/core/class.session.php

```diff
@@ -26,6 +26,9 @@ class Gdn_Session {
     /** @var object A User object containing properties relevant to session */
     public $User;
 
+    /** @var Gdn_SQLDriver Contains the sql driver which is set in method stash. */
+    public $sql;
```

Comment:
I hope it's the last thing I have to bother you with before I change my PR: should it be `protected $sql` or should I make it   `protected $_sql`?